app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div>
+  <h2 class="form-title">Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, placeholder: "Name"  %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autocomplete: "email", placeholder: "Email Address" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password" %>
+    <% if @minimum_password_length %>
+    <em class="info">(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+  </div>
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure that the `devise` gem is correctly setup. Currently, I'm getting this error when I try to create a new user:

![image](https://github.com/Lucash2022/Blog_App/assets/77069011/c6ac1e3f-d735-4c3a-9ce5-db183ba19692)

This error happens because you're sending a confirmation mail but haven't set up a mailer default url. According to the [`devise` gem official docs](https://github.com/heartcombo/devise#getting-started):

![Screenshot from 2023-06-27 10-08-05](https://github.com/Lucash2022/Blog_App/assets/77069011/61b54b6b-dd60-49af-a8ec-1db7192175f5)

Please consider fixing this! 🤓 🔧

Reply:
@anagudelogu Thank you for the review. We have made the change at this [commit](https://github.com/Lucash2022/Blog_App/pull/12/commits/4d805f1b3c3c452399c87ef8220b899b4a5dd601)